### PR TITLE
 added cart card

### PR DESCRIPTION
**Pull request by dendoesit**
## Describe your changes

Added cart card in storybook

## Justify why they are needed

For the cart page.

### CODE REVIEW

File: packages/ui/src/components/CartCard/CartCard.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+
+const ProductCard = styled.div({
+  padding: '0 1rem 1.5rem 1rem',
```
Comment (robinandeer) [inline packages/ui/src/components/CartCard/CartCard.tsx:5]: We have spacing tokens in the theme now :)

File: packages/ui/src/components/CartCard/CartCard.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  display: 'block',
+  width: '100%',
+  alignItems: 'center',
+  fontSize: '1rem',
+})
+
+const TitleElement = styled.div({})
+const IconElement = styled.div(
+  {
+    backgroundColor: colorsV3.gray300,
```
Comment (robinandeer) [inline packages/ui/src/components/CartCard/CartCard.tsx:22]: Get these directly from the theme instead like you do below in the border style!

File: packages/ui/src/components/CartCard/CartCard.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+const RemoveButton = styled.button({
+  marginTop: '2rem',
+  textDecoration: 'underline',
+  fontSize: '0.925rem',
+})
+
+export type CartCardProps = {
+  title?: string
+  price?: string
```
Comment (robinandeer) [inline packages/ui/src/components/CartCard/CartCard.tsx:43]: Hehe how are they both optional? What's the point of rendering it without any of that information? 😅

**Review by robinandeer — COMMENTED** [review]
Great job! Although this seems more like a store component rather than something for the UI kit, don't you think?

I would put it under the store app for now.

File: packages/ui/src/components/CartCard/CartCard.text.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { screen } from '@testing-library/react'
```
Comment (guilhermespopolin) [inline packages/ui/src/components/CartCard/CartCard.text.tsx:1]: I believe that file should be called `CartCard.test.tsx`, right?! 😄

**Review by guilhermespopolin — APPROVED** [review]
Nice :)
Just added a comment about the test file and I agree with Robin about `CartCard` into `store`.

File: packages/ui/src/components/CartCard/CartCard.tsx (hunk 3 above)
Comment (dendoesit) [reply packages/ui/src/components/CartCard/CartCard.tsx:43]: oops, wanted only the price to be optional 😄

File: packages/ui/src/components/CartCard/CartCard.tsx (hunk 1 above)
Comment (dendoesit) [reply packages/ui/src/components/CartCard/CartCard.tsx:5]: is there a way we can do the same ?

File: packages/ui/src/components/CartCard/CartCard.text.tsx (hunk 4 above)
Comment (dendoesit) [reply packages/ui/src/components/CartCard/CartCard.text.tsx:1]: 😂

**Pull request merged**